src/coreclr/src/gc/gc.cpp

```diff
@@ -31998,6 +32053,19 @@ size_t gc_heap::desired_new_allocation (dynamic_data* dd,
             cst = min (1.0f, float (out) / float (dd_begin_data_size (dd)));
 
             f = surv_to_growth (cst, limit, max_limit);
+            if (conserve_mem_setting != 0)
+            {
+                // if this is set, compute a growth factor based on it.
+                // example: a setting of 6 means we have a goal of 60% live data
+                // this means we allow 40% fragmentation
+                // to keep heap size stable, we only use half of that (20%) for new allocation
+                // f is (live data + new allocation)/(live data), so would be (60% + 20%) / 60% or 1.33
+                float f_conserve = (10.0f / conserve_mem_setting - 1) * 0.5f + 1.0f;
```

Comment:
nit - ()

                `float f_conserve = ((10.0f / conserve_mem_setting) - 1) * 0.5f + 1.0f;`

this is what you had for framework :D

Reply:
I see - added the extra pair of parentheses...